### DISCUSSION TITLE
Alfen Eve NG910 Solar Charging

### DISCUSSION THREAD

**Discussion by ROosterloo1988** [leeyuentuen/alfen_wallbox#45] [Q&A] [answered]
Hello,

Can anyone help me with a good setup for solar charging? My wallbox is setup and controlled at Eneco but i have also access to the wallbox by app. 

My situation:

Input 3 phase 25A
27 solarpanels (10.660kwp)
Alfen Eve NG910 (firmware 6.3.0-4185)
Loadbalanced by Eneco with P1 extenders
With this maximum of 11KWh.

When i have selected solar charging and active loadbalanced by p1. The wallbox is charging with 4.5 KWh on 1 phase.

Greetz Ronald

**Comment by leeyuentuen** [comment]
i think that you can max 3.5-4.1 kwh on 1 phase. if it goes higher, you need to let them change to 3 phase to get them on 11kwh

**Reply by ROosterloo1988** [reply]
It is setup to charge on 3 phase.

**Comment by ROosterloo1988** [comment, marked as answer]
It works for now.

In the app and the service installer i can't select solar charging on the eneco profile loadbalanced by serial.

In HACS with your plugin it works!

Many thanks!
